_rules/SC4-1-2-aria-attribute-has-valid-value.md

```diff
@@ -0,0 +1,256 @@
+---
+name: ARIA attribute has valid value
+test_type: atomic
+
+description: |
+   This rule checks that all ARIA 1.1 attributes (`aria-*`) have valid values
+   
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any [non-empty](#non-empty) attribute that is placed on an HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies), and where the attribute name starts with `aria-` and where the attribute can be found in [WAI-ARIA 1.1, Definitions of States and Properties](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def).
```

Comment:
I don't think this meets the "plain language" requirement. We should probably break this up or list it out in some way to make this easier to read.

I don't think that "exposed to assistive technologies" should be part of this. There are a number of ways that ARIA on elements that are not exposed to assistive technologies will impact users. For example, anything referenced through aria-labelledby does not have to be exposed to assistive technologies in order for it to become part of the accessible name.